docs/newbs_getting_started.md

```diff
@@ -118,6 +118,13 @@ NOTE: remember to follow the instructions printed at the end of installation (us
 
 <!-- tabs:start -->
 
+Tips for FreeBSD 13.0:
+
+I'm on FreeBSD 13.0 and I've had two problems:
+py37-setuptools was a tad too old, I had version 44 and qmk cli needed 45, so I downloaded a newer wheel (57) from pipy and installed it.
+Once all the python dependencies went through, the cli wouldn't work, due to library issues. I resolved it by completely uninstalling python hid packages I had, and compiling one from source.
+I downloaded hid-1.0.4.tar.gz, ran python setup.py build, then python setup.py install and my cli started working...
+
```

Comment:
This location is incorrect, it is not under a heading so does not render.

Changes should be made somewhere near,
https://github.com/qmk/qmk_firmware/blob/8bba77db0da2aa189103ef9498780d2c915ad4e9/docs/newbs_getting_started.md#L113
or
https://github.com/qmk/qmk_firmware/blob/8bba77db0da2aa189103ef9498780d2c915ad4e9/docs/newbs_getting_started.md#L157

Also the wording is far too personal, and inconsistent with the rest of the document

```suggestion
```

Reply:
Fixed. I used the original doc as a reference and now I understand the sections and tabs.
I decided to place this under the "run qmk setup" section - FreeBSD tab.